configCivcraft.yml

```diff
@@ -11000,11 +10981,11 @@ recipes:
         amount: 128
       granite:
         material: STONE
-        amount: 64
```

Comment:
Like I told you in our reddit discussion, reinforcement rebalance is already in the works and will not involve direct player led contribution, as it requires administrative knowledge (similar to ore distribution).